src/runtime_src/core/common/drv/kds_cu_ctrl.c

```diff
@@ -52,9 +58,10 @@ add_ctx(struct kds_cu_ctrl *kcuc, struct kds_client *client,
 	}
 
 	/* Remove OPs from flag */
-	info->flags &= ~CU_CTX_OP_MASK;
-	shared = (info->flags != CU_CTX_EXCLUSIVE);
+	prop = info->flags & CU_CTX_OP_MASK;
+	shared = (prop != CU_CTX_EXCLUSIVE);
```

Comment:
I'm confused, prop contains only CU_CTX_OP_xxx now, but CU_CTX_EXCLUSIVE is not part of it, why do you compare them?

Reply:
Oh.. It should be CU_CTX_PROP_MASK.